sample-react-app/README.md

```diff
@@ -0,0 +1,84 @@
+[![Website](https://img.shields.io/badge/View-Website-blue)](https://sample-project.s3-web.us-east.cloud-object-storage.appdomain.cloud)
+
+# Getting Started with Sample React App
+
+Sample React App is a simple react web app that can be used as a template/starting point for working with React Web framework. 
+
+Instructions to run the Sample React App;
+1. [Local Development](#run-the-sample-react-app-for-local-development)
+2. [IBM Cloud](#run-the-sample-react-app-on-IBM-cloud)
+
+### Run the Sample React App for local development
+#### Pre-requisites
+1. [Install Node and NPM](https://nodejs.org/en/download/)
+
+#### Build the App
+- Start a terminal/CMD in `~/Project-Sample/sample-react-app` directory.
+- Execute command `npm install`.
+
+#### Run the App
+- Execute command `npm start`.
+- Open a web browser and enter `http://localhost:3000`.
+
+#### Run App tests
+- Execute command `npm test`.
+
+### Run the Sample React App on IBM Cloud
+#### Pre-requisites
+1. IBM Cloud Account:
+    - [Create free IBM Cloud account](https://cloud.ibm.com/registration)
```

Comment:
Let's use the CFC community registration link since it has better credits: https://developer.ibm.com/callforcode/get-started/

Reply:
The free IBM Cloud account registration link is updated with the CFC community registration link